src/js/Analytics.js

```diff
@@ -0,0 +1,180 @@
+'use strict';
+
+const Analytics = function (trackingId, userId, appName, appVersion, gitRevision, os, checkForDebugVersions, optOut, debugMode, buildType) {
```

Comment:
Please use modules for new code and import where appropriate.

`strict` is not needed when using modules it's a default mode.

Reply:
Need to address this